collections/_development_standards/new-solutions.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title:  "New Solutions"
+category: development
+---
+
+## New solutions
+
+Below are some considerations that we should be taking into account when creating new solutions.
+
+### Which .Net version should I use
+
+In the main new solutions should be using the latest long-term support version available at the time. It also needs to be supported by our build servers so a check-in with the DevOps team before starting is always useful.
+
+### Storage mechanisms
+
+Azure SQL or CosmosDB, Is the data likely to be structured or unstructured, will it require fast reads and horizontal scalability?
```

Comment:
For me our default should be SQL storage for most applications. There are scenarios where using unstructured storage is appropriate but we have several examples in the past where teams wanted to use an 'unstructured data storage solution because it faster'. In reality they just wanted to move the data versioning problem out of their scope. I think it needs to be a wider discussion and agreement (inc Arch) if application data isn't going to be stored in SQL.

It needs a decision which is not centred around the teams immediate scope (especially if other teams are impacted).

Reply:
@DavidBSteele - If this page was not just for developers but everyone in the team involved with new software creation, what wording would you put around this topic from an architectural point of view that you would want the team to know to guide them to the correct choice?

Reply:
We need some arch pages to cover this fully...but we haven't got them yet.

How about this as a placeholder? Happy for changes...if you don't like it.

"The default storage for new service implementations should be SQL storage for most applications. There are scenarios where using unstructured storage is appropriate but this must be agreed by the Architect and DevOps before an alternative is selected. Clear reasons why its more appropriate must be established. Statements like 'I want to use an unstructured data storage solution because it faster' are not adequate. 

Unstructured data storage makes some key decisions about versioning of the data that you are storing which will have an impact outside of the team's (i.e. Reporting). This will often simply move the data versioning problem out of one teams scope and into another. 

Having said this there are scenarios that unstructured data storage is the right approach. If you have a use case that you think needs unstructured data storage then a discussion including the Architect and DevOps is required."

Reply:
I've updated the wording